src/NServiceBus.RavenDB/NServiceBus.RavenDB.csproj

```diff
@@ -11,11 +11,11 @@
   <ItemGroup>
     <PackageReference Include="Fody" Version="2.2.1" PrivateAssets="All" />
     <PackageReference Include="Janitor.Fody" Version="1.5.2" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-rc0001, 8.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
     <PackageReference Include="Obsolete.Fody" Version="4.3.2" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
-    <PackageReference Include="RavenDB.Client" Version="[3.5.5-patch-35245, 3.6.0)" />
+    <PackageReference Include="RavenDB.Client" Version="[3.5.5, 3.6.0)" />
```

Comment:
@bording Package on the lowest that works, tests on the latest - that's what you're going for, right?

Reply:
Correct.